content/en/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -23,6 +23,12 @@ those network endpoints can be routed closer to where it originated.
 For example, you can route traffic within a locality to reduce
 costs, or to improve network performance.
 
+{{< note >}}
+
+Currently this feature is only viable for large scale workloads, since the `auto` heuristic requires you to constantly have a proportional number of endpoints based on the amount of allocatable CPU in your cluster per zone (for all nodes in that zone), as your cluster grows, so must your endpoints grow.
+This can lead to situations where if your endpoints don't scale fast enough, topology hints will be removed, this can potentialy lead to additional data transfer costs.
```

Comment:
For reference, I created a [simple spreadsheet](https://docs.google.com/spreadsheets/d/1_4QyfQEcnEWYpcmUc1Q6cjgkLMhMqpTgok2UUpOmUhE/view?usp=sharing&resourcekey=0-k7XjENYqtAyBfm12lDq0KQ) that shows which numbers of endpoints would result in hints being allocatable in a cluster with 3 equivalent zones. In that case, hints would not be allocatable at 1, 2, 4, 5, 8, and 11 endpoints. Of course clusters come in many shapes and sizes, so this is only 1 example of how it would work.

Reply:
Seems fine to me